.travis.yml

```diff
@@ -0,0 +1,58 @@
+language: python
+python:
+  - "3.7"
+
+branches:
+  only:
```

Comment:
maybe we should add a regular expression branch, something like `/^tobetested-.*$/` To be able to run tests using a pattern

Reply:
Good idea, but I don't think that in practice people will remember to name their branch correctly. Also, in new branches, devs can override the `branches` property to allow Travis builds for their branch.

Reply:
Ok thank you for the details @AurelienGasser